modular_stonehedge/licensed-dreamkeep/LICENSE.md

```diff
@@ -0,0 +1,5 @@
+# LICENSE
+All the items in this folder and its subdirectories are licensed by Dreamkeep as a collaborative work between it's artists, additional credit may be given within the folder.
+
+Redistribution, reproduction, or modification of these assets is prohibited without express permission from both licensors. These assets are for use solely within this project and may not be extracted or repurposed for other projects or commercial use.
```

Comment:
I'm not sure if this is valid under the other licenses, it depends on what "modification" or "redistribution" means, I guess. Mainly: The assets are referenced in code, meaning it will fail to compile if the assets are removed. If you replace the files with blank ones, that might be considered modification. If this is a covert way to prevent people from porting the associated code, that doesn't work because GPLv3 explicitly forbids further restrictions on who can use code licensed under it.

Reply:
those licenses cover only the original icons and sprites which TG does not license those.